components/search/search.tsx

```diff
@@ -70,6 +72,55 @@ const determineVisibleSearchHintOptions = (searchTerms: string, searchType: Sear
     return newVisibleSearchHintOptions;
 };
 
+export const formatRecentSearch = (searchParams: SearchParams) => {
+    const searchQueryItems = [];
+    if (searchParams.terms) {
+        searchQueryItems.push(searchParams.terms);
+    }
+    if (searchParams.excluded_terms) {
+        searchQueryItems.push(`-${searchParams.excluded_terms}`);
+    }
+    if (searchParams.on_date) {
+        searchQueryItems.push(`on:${searchParams.on_date} `);
```

Comment:
A few of these still have the trailing space that isn't needed any more now that we're joining the items together

Reply:
My bad, I missed a few of them :)